src/App.tsx

```diff
@@ -21,9 +38,27 @@ const App = () => {
   const [drawerOpen, setDrawerOpen] = useState(isGreaterThanLgBreakpoint());
 
   useEffect(() => {
-    const isAccessTokenPresent = getAuthSession('accessToken') !== null;
-    const isTokenExpired = checkAuthStatusService();
-    setAuthenticated(isTokenExpired || isAccessTokenPresent);
+    const checkAuth = async () => {
+      const isAccessTokenPresent = getAuthSession('accessToken') !== null;
+      const isTokenAlive = checkAuthStatusService();
+      let isSessionAlive = false;
+      if(!isAccessTokenPresent){
```

Comment:
Code formatting is incorrect. @Snehil-Shah Can you please check your code editor configuration?